test/testleakautovar.cpp

```diff
@@ -2762,6 +2763,19 @@ class TestLeakAutoVar : public TestFixture {
         ASSERT_EQUALS("[test.cpp:3] -> [test.cpp:4]: (error) Returning/dereferencing 'p' after it is deallocated / released\n", errout_str());
     }
 
+    void return11() { // #13098
+        const Settings s = settingsBuilder().library("std.cfg").build();
```

Comment:
The other tests don't explicitly load std.cfg.

Reply:
Ah, I'll fix that. I thought it was required but it was really `cpp = true` that was required.